com.unity.render-pipelines.universal/Editor/Camera/UniversalRenderPipelineCameraEditor.cs

```diff
@@ -891,29 +892,36 @@ void DrawTargetTexture(UniversalRenderPipelineAsset rpAsset)
 
         void DrawVolumes()
         {
-            bool hasChanged = false;
-            LayerMask selectedVolumeLayerMask;
-            Transform selectedVolumeTrigger;
-            if (m_SerializedCamera == null)
```

Comment:
This is the only place in this script that checks if the serialized camera is null. 
I'm removing this check because if it is null then we would have numerous errors before this one triggering.